ffmpeg/ffmpeg_errors.go

```diff
@@ -49,6 +49,34 @@ func error_map() map[int]error {
 
 var ErrorMap = error_map()
 
+func non_retriable_errs() []string {
+	errs := make([]string, 0, 20)
+	// Add in Cgo LPMS specific errors
+	for _, v := range lpmsErrors {
+		errs = append(errs, v.Desc)
+	}
+	// Add in internal FFmpeg errors
+	// from https://ffmpeg.org/doxygen/trunk/error_8c_source.html#l00034
+	ffmpegErrors := []string{
+		"Decoder not found", "Demuxer not found", "Encoder not found",
```

Comment:
Curious what the criteria for selecting these specific errors was since I see that there are a lot more documented in the above link?

Reply:
No specific criteria - picked the few that immediately seemed related to bad input/parameters. I've personally only seen "Invalid argument" often.

Plan was to iterate upon this list if we see particular internal errors propping up in future that can be classified as retriable. WDYT?